spec/fixtures/build.gradle.kts

```diff
@@ -34,6 +34,11 @@ dependencies {
 
     // Use the Kotlin JUnit integration.
     testImplementation(     "org.jetbrains.kotlin:kotlin-test-junit:1.0.0"     )
+
+    implementation(project(":my-internal-lib"))
```

Comment:
May add support for these later, depending on whether we want to consider projects as "packages" (i.e. they show up as a node in the dep tree From `gradle dependencies -q`)